VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -457,12 +457,20 @@ export abstract class WaitingRoomBaseDirective {
         );
 
         this.eventHubSubscription$.add(
-            this.eventService.getParticipantAdded().subscribe(async participantAddedMessage => {
-                this.logger.debug(`[WR] - Participant Added`, participantAddedMessage);
-                this.notificationToastrService.showParticipantAdded(
-                    participantAddedMessage.participant,
-                    this.participant.status === ParticipantStatus.InHearing
+            this.eventService.getParticipantsUpdated().subscribe(async participantsUpdatedMessage => {
```

Comment:
This is fine, I don't think it should be a change for this PR. But we could consider moving this logic to participant service and adding a OnParticipantAdded observable to the participant service, then handle removed etc.

Reply:
That would be preferable yes. Is there a tech debt story for refactoring this component?